pkg/operator/ceph/object/spec.go

```diff
@@ -678,3 +705,97 @@ func (c *clusterConfig) rgwTLSSecretType(secretName string) (v1.SecretType, erro
 func getDaemonName(rgwConfig *rgwConfig) string {
 	return fmt.Sprintf("ceph-%s", generateCephXUser(rgwConfig.ResourceName))
 }
+
+func (c *clusterConfig) sseKMSDefaultOptions() []string {
+	if kmsEnabled, _ := c.CheckRGWKMS(); kmsEnabled {
```

Comment:
These methods should take the enabled options as parameters so we can avoid swallowing any errors.

Reply:
Actually this check is reductant, I will remove those

Reply:
This check was added to work with unit test TestAWSServerSideEncryption() so that this functions desired options in different scenarios. The check is reluctant for actual code workflow.

Reply:
Not sure what you're saying, but we need to handle the error
```suggestion
	if kmsEnabled, err := c.CheckRGWKMS(); kmsEnabled {
```

Reply:
Yes. We should either handle the error or take `kmsEnabled` as an input parameter (preferred).